kobo/apps/hook/models/service_definition_interface.py

```diff
@@ -113,7 +116,15 @@ def send(self):
                     text = response.text
                     status_code = response.status_code
                 self.save_log(status_code, text)
-
+            except SSRFProtectException as e:
```

Comment:
I should've brought this up way back in #1864, but these very large `try` blocks and catch-call `except Exception as e` handlers should be refactored at some point. In this `send()` method, for example, I think the only calls that should be inside `try`s are:
* `SSRFProtect.validate(self._hook.endpoint)`
* `response.raise_for_status()`

I'm not sure why `except Exception as e` is better than letting the exception fall through to Sentry. Maybe I'm forgetting something.